packages/babel-preset-env/src/normalize-options.js

```diff
@@ -218,19 +212,19 @@ export function normalizeCoreJSOption(
   let rawVersion;
 
   if (useBuiltIns && corejs === undefined) {
-    rawVersion = 2;
+    rawVersion = "3.6";
```

Comment:
We should also convey this change in the release plan, that if you specify `useBuiltIns` but leave `corejs` empty, it will now defaults to `3.6`.